### ISSUE TITLE
Wrong suggestion for tag 'short'

### ISSUE THREAD

**Issue by iiitomo** [osm-fr/osmose-backend#1578]
When detecting tag 'short', Osmose currently proposes to replace it by tag 'sport'.
See example below.
Instead it should propose to replace it by tag 'short_name' (or 'alt_name').

![osmose_bad_suggestion_for_short](https://user-images.githubusercontent.com/58949423/192235299-770b5558-8fed-4c89-8492-5ecba2d00072.PNG)

**Comment by frodrigo** [comment]
As undocumented tag with de facto quasi no-usage, yes it is good report from Osmose.

Note it is a typo check. So it suggest fixing a typo. There is also other controls, like deprecated or tagcleaner, to replace tags.

**Comment by iiitomo** [comment]
Hi frodrigo, I don't understand your answer.
I've fixed a dozen of 'short' tags by 'short_name', because values were clearly corresponding to a short way of writing the name). None of these cases was a typo nor corresponded to 'sport'.
short_name tag is documented in https://wiki.openstreetmap.org/wiki/Key:name
It might not be very used, but is very likely the best replacement that Osmose can suggest for 'short'.

**Comment by iiitomo** [comment]
I see you've closed several of my other reports for same reason, so I think that for these tags the typo_check control should be disabled so that Osmose propose correct suggestion via other controls. Alternatively, make Typo_check lower Severity, so that correct suggestion comes first.
(Currently Typo_check is Severe, so it comes first)
